crates/axvm/src/vm.rs

```diff
@@ -19,17 +20,26 @@ use crate::{has_hardware_support, AxVMHal};
 const VM_ASPACE_BASE: usize = 0x0;
 const VM_ASPACE_SIZE: usize = 0x7fff_ffff_f000;
 
+// Todo: should Vcpu related type be put into `axvcpu`` crate?
```

Comment:
Definitely yes.

Reply:
But `AxArchVCpuImpl` is in `axvm`

Reply:
`axvm` deps `axvcpu`, so it won't be a problem?

Reply:
Yes, it will be a problem, so we cannot move this Vcpu related type into `axvcpu` crate